scripts/clock.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import print_function
+
+from django.core.management import call_command
+
+from apscheduler.schedulers.blocking import BlockingScheduler
+
+from bedrock.events.cron import cleanup_ical_events, update_ical_feeds
+from bedrock.mozorg.cron import update_tweets
+from scripts import update_firefox_os_feeds, update_tableau_data
+
+
+schedule = BlockingScheduler()
+
+
+class scheduled_job(object):
+    """Decorator for scheduled jobs. Takes same args as apscheduler.schedule_job."""
+    def __init__(self, *args, **kwargs):
+        self.args = args
+        self.kwargs = kwargs
+
+    def __call__(self, fn):
+        job_name = fn.__name__
+        self.name = job_name
+        self.callback = fn
+        schedule.add_job(self.run, id=job_name, *self.args, **self.kwargs)
+        return self.run
+
+    def run(self):
+        self.log('starting')
+        try:
+            self.callback()
+        except Exception as e:
+            self.log('CRASHED: {}'.format(e))
+            raise
+        else:
+            self.log('finished successfully')
+
+    def log(self, message):
+        print('Clock job {0}: {1}'.format(self.name, message))
+
+
+@scheduled_job('interval', seconds=5)
+def job_should_pass():
```

Comment:
This one and the next one are obviously not for prod. They should help with testing what things look like in papertrail and whether things work or not at all :)